message MSG_CLIENT_DM_DOMAIN_DESTROY
{
    optional string token = 1;
    optional uint32 domainID = 2;
}

message MSG_DM_CLIENT_DOMAIN_DESTROY_ACK
{
    optional int32 statusCode = 1;
}